src/components/notification-center/NotificationCenter/styles.module.css

```diff
@@ -13,6 +13,13 @@
   border: 1px solid var(--color-border-light);
 }
 
+@media only screen and (max-width: 470px) {
```

Comment:
We are using MUI in this project and try to adhere to the default breakpoints as much as possible. In this case I would suggest to use the max-sm breakpoint which is 599.95px.